packages/bottender/src/slack/SlackEvent.ts

```diff
@@ -286,6 +300,22 @@ export default class SlackEvent implements Event<SlackRawEvent> {
   get isBotMessage(): boolean {
     return (this._rawEvent as any).subtype === 'bot_message';
   }
+
+  /**
+   * Determine if the event is a slash command event.
+   *
+   */
+  get isSlashCommand(): boolean {
+    return !!(this._rawEvent as SlashCommandEvent).command;
+  }
+
+  /**
+   * The slash command name.
+   *
+   */
+  get slashCommand(): string | null {
```

Comment:
We should use `command` and `isCommand` consistently. It seems that slack doesn't use slash in their code or field names.